sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -1056,8 +1290,30 @@ func (m *networkFlowManager) UnregisterCollector(hostname string, sequenceID int
 }
 
 func (h *hostConnections) Process(networkInfo *sensor.NetworkConnectionInfo, nowTimestamp timestamp.MicroTS, sequenceID int64) error {
-	updatedConnections := getUpdatedConnections(h.hostname, networkInfo)
-	updatedEndpoints := getUpdatedContainerEndpoints(h.hostname, networkInfo)
+	flowMetrics.NetworkConnectionInfoMessagesRcvd.With(prometheus.Labels{"Hostname": h.hostname}).Inc()
+	now := timestamp.Now()
+
+	for _, updatedConnection := range networkInfo.UpdatedConnections {
+		if updatedConnection.CloseTimestamp == nil {
+			flowMetrics.IncomingConnectionsEndpoints.With(prometheus.Labels{"object": "connections", "closedTS": "unset"}).Inc()
+		} else {
+			flowMetrics.IncomingConnectionsEndpoints.With(prometheus.Labels{"object": "connections", "closedTS": "set"}).Inc()
+		}
+	}
+	for _, updatedEp := range networkInfo.UpdatedEndpoints {
+		if updatedEp.CloseTimestamp == nil {
+			flowMetrics.IncomingConnectionsEndpoints.With(prometheus.Labels{"object": "endpoints", "closedTS": "unset"}).Inc()
+		} else {
+			flowMetrics.IncomingConnectionsEndpoints.With(prometheus.Labels{"object": "endpoints", "closedTS": "set"}).Inc()
+		}
+	}
```

Comment:
I think we can avoid looping twice over `UpdatedConnections` and UpdatedEndpoints` if we have this block moved to `getUpdatedConnections` and `getUpdatedContainerEndpoints`.

If you decided not to move this block:
nit: use `GetUpdatedConnections() and `GetUpdatedEndpoints()` instead of accessing directly the struct fields.